Common_Words_Scraping.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+import time
+from lxml import html
+import requests
+import re
+from bs4 import BeautifulSoup
+import os
+import csv
+from Wiktionary_scraper import *
+
+def common_words(language):
```

Comment:
What about making a class  to encapsulate those methods ? This might help isolating components.